### ISSUE TITLE
Experiments Import job is failing

### ISSUE THREAD

**Issue by sunahsuh** [mozilla/experiments-viewer#198]
We finally got experiment aggregates back up and running but the import step failed:

```Inserting data for experiment: pref-flip-screenshots-1369150
Traceback (most recent call last):
  File "/mnt/analyses/./import.py", line 240, in <module>
    'total_pings', total_pings)
  File "/mnt/analyses/./import.py", line 191, in create_stat
    cursor.execute(sql, params)
  File "/mnt/anaconda2/lib/python2.7/site-packages/psycopg2/extras.py", line 429, in execute
    return super(LoggingCursor, self).execute(query, vars)
psycopg2.ProgrammingError: relation "api_stats" does not exist
LINE 1: INSERT INTO api_stats (dataset_id, metric_id, population, su...```

**Comment by sunahsuh** [comment]
Seems like the prod deployment is behind and the migration with the stats table hasn't been run. Unfortunately, robhudson is out today and tomorrow but @rafrombrc is trying to get a deployment done.

**Comment by robhudson** [comment]
I ran the migrations on production and cleaned up dead imports and the latest imports have been run as of today.

**Comment by sunahsuh** [comment]
To recap:

Friday:
- The latest migration creating the `api_stats` table hadn't run on production
- The first run of the import script after migration failed due to a missing cascading delete
- We merged in #204 and deployed it to prod to fix that issue
- The migration adding the cascading delete wasn't applied in the deployment
Saturday:
- @robhudson caught up on backscroll after PTO and ran the migration
